format.sh

```diff
@@ -38,11 +38,22 @@ source venv/bin/activate
 
 echo "Checking Python files..."
 if [ "$COMMAND" == "fix" ]; then
-  black .
+  black --quiet .
 else
-  black --check --diff .
+  black --quiet --check --diff .
```

Comment:
This line is executed in the CI, and it's easier to have the file list to fix printed out at the end of the process.

```suggestion
  black --check --diff .
```